lib/linguist/languages.yml

```diff
@@ -4564,6 +4564,7 @@ Shell:
   - ".tool"
   - ".zsh"
   filenames:
+  - ".bash_aliases"
```

Comment:
Could you please also add an entry for this without the dot?

~~~diff
   - 9fs
   - PKGBUILD
+  - bash_aliases
   - bash_logout
   - bash_profile
~~~